scripts/Diffraction/isis_powder/routines/focus.py

```diff
@@ -48,22 +48,38 @@ def _focus_one_ws(input_workspace, run_number, instrument, perform_vanadium_norm
         summed_empty = common.generate_summed_runs(empty_sample_ws_string=run_details.sample_empty,
                                                    instrument=instrument,
                                                    scale_factor=instrument._inst_settings.sample_empty_scale)
-    if summed_empty is not None:
-        input_workspace = common.subtract_summed_runs(ws_to_correct=input_workspace,
-                                                      empty_sample=summed_empty)
-
-    # Crop to largest acceptable TOF range
-    input_workspace = instrument._crop_raw_to_expected_tof_range(ws_to_crop=input_workspace)
-
-    # Correct for absorption / multiple scattering if required
-    if absorb:
-        input_workspace = instrument._apply_absorb_corrections(run_details=run_details, ws_to_correct=input_workspace)
+    absorb_method = None
+    try:
+        absorb_method = instrument._inst_settings.absorb_method
+    except AttributeError:
+        absorb_method = 'Mayers'
```

Comment:
If you reinstated the `absorb_method` parameter on all the instruments would it remove the need for this? That might be nicer than doing this here.
The NotImplementedError inside abstract_inst.py would effectively turn off the PP feature on all the other instruments which is fine